neo/core/irregularlysampledsignal.py

```diff
@@ -353,7 +355,7 @@ def mean(self, interpolation=None):
         else:
             raise NotImplementedError
 
-    def resample(self, at=None, interpolation=None):
+    def resample_irregularly(self, at=None, interpolation=None):
```

Comment:
What is the thinking behind providing a not-implemented method? Wouldn't it be better just to remove this?

Reply:
I guess the idea was to make a suggestion for the future interface, so I kept it. But I am happy to remove it if you agree.

Reply:
I think it would be better to remove it to avoid confusing users - but perhaps you could open an issue so as to remind us of the idea?